### PR TITLE
Android audio track

### PR DESCRIPTION
**Pull request by yasinugrl**
I've added the possibility to use custom audioTracks on android part. As you know that you cant add custom audio track in code part on Android. You just select audio track that whit in video chanel. But I have need custom mp3 file add in code part. So in IOS part @reime005 done it by using **alternativeAudioSource**. Thanks a lot for that. Developed android part by using audioTracks props by me. Now You can add custom audio file in array like textTracks. So just add like this: 

```
 audioTracks={[
              {
                language: 'en',
                title: 'English CC',
                type: 'audio/mp4a-latm',
                uri: "http://webaudioapi.com/samples/audio-tag/chrono.mp3"
              },
              {
                language: 'en',
                title: 'English CC',
                type: 'audio/mp4a-latm',
                uri: "https://www.soundhelix.com/examples/mp3/SoundHelix-Song-1.mp3"
              }
            ]}
```

In summary, You can use _alternativeAudioSource_ for custom audio in IOS and you can use audioTracks in Android.

instalition with yarn:
`yarn add  yasinugrl/react-native-video#androidAudioTrack`

with npm:
`npm i reime005/react-native-video#audioTrack --save`

### CODE REVIEW

**Comment by reime005** [comment]
Thanks for that! I've used the workaround of adding the 2nd audio track to the video(s), using ffmpeg. That's better now
